projects/plugins/jetpack/modules/module-info.php

```diff
@@ -723,6 +723,25 @@ function jetpack_custom_jetpack_manage() {
 }
 add_action( 'jetpack_module_more_info_manage', 'jetpack_custom_jetpack_manage' );
 
+/**
+ * Post list info.
+ */
+function jetpack_post_list_link() {
+	echo esc_url( Redirect::get_url( 'jetpack-support-post-list' ) );
+}
+add_action( 'jetpack_learn_more_button_post-list', 'jetpack_post_list_link' );
+
+/**
+ * Post List description.
+ */
+function jetpack_post_list_info() {
+	esc_html_e(
+		'Display extra information alongside each post in your dashboard's Posts screen.',
```

Comment:
Oh, I messed up with that apostrophe, my bad.

```suggestion
		'Display extra information alongside each post in your dashboard’s Posts screen.',
```